test/unit/prisma-photon/actual.js

```diff
@@ -0,0 +1,4 @@
+[
+  "test/unit/prisma-photon/input.js",
+  "test/unit/prisma-photon/node_modules/@generated/photon/index.js"
+]
```

Comment:
This file shouldn't be committed

Reply:
Thanks! `actual.js` should then probably be added to `.gitignore`.